src/client/components/Overview/OverviewSearchMessagesBtn.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState, useEffect } from 'react';
+
+function messagesRequest() {
+  return new Promise((resolve) => setTimeout(resolve, 2000));
+}
+
+function OverviewSearchMessagesBtn() {
+  const [isLoading, setLoading] = useState(false);
+
+  useEffect(() => {
+    if (isLoading) {
+      messagesRequest().then(() => {
+        setLoading(false);
+      });
+    }
+  }, [isLoading]);
+
+  const handleClick = () => setLoading(true);
+
+  return (
+    <button
+      className="OverviewSearchMessagesBtn"
+      isLoading={isLoading}
+      type="submit"
+      disabled={isLoading}
+      onClick={!isLoading ? handleClick : null}
+    >
+      {isLoading ? 'Loading…' : 'Chats'}
+    </button>
+  );
+}
+
+export default OverviewSearchMessagesBtn;
```

Comment:
What is this? Lucy has done the autocomplete and the search button

Reply:
It is a button for the Chats which located at the bottom of mockup 1. It is not working because it should be connected to the messages / chats.I don't know who is working on that.